docs/source/docker/integration/docker_with_datasource.rst

```diff
@@ -16,8 +16,13 @@ the following workloads in one single run:
 - compute the metadata of the images
 - create a dataset in the Lightly Platform from the sampled subset
 
-Support for streaming from Google Cloud Storage and Azure Blob Storage is
-planned. If you need any of these, write us so that we prioritize implementing it.
+Streaming from Google Cloud Storage and Azure Blob Storage is also supported.
+Please follow the
+`Google Cloud Storage <https://docs.lightly.ai/getting_started/dataset_creation/dataset_creation_gcloud_bucket.html`_
+ or the
+`Azure Blob Storage <https://docs.lightly.ai/getting_started/dataset_creation/dataset_creation_azure_storage.html>`_
+ guide to learn how to setup your bucket and configure a datasource.
+
```

Comment:
GCS and Azure support is not yet implemented in the docker :(  There seems to be an issue for M0.1 but it is not in the cycle: https://linear.app/lightly/issue/LIG-770/make-compute-worker-docker-work-with-azure-datasource

Reply:
ach :(